[2025-09-12T19:10:24.046+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-12T19:10:24.165+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.send_forecast scheduled__2025-09-11T16:00:00+00:00 [queued]>
[2025-09-12T19:10:24.189+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.send_forecast scheduled__2025-09-11T16:00:00+00:00 [queued]>
[2025-09-12T19:10:24.191+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 3
[2025-09-12T19:10:24.232+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): send_forecast> on 2025-09-11 16:00:00+00:00
[2025-09-12T19:10:24.252+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'send_forecast', 'scheduled__2025-09-11T16:00:00+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmp4vz2ryu8']
[2025-09-12T19:10:24.257+0000] {standard_task_runner.py:105} INFO - Job 209: Subtask send_forecast
[2025-09-12T19:10:24.258+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=15933) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-12T19:10:24.260+0000] {standard_task_runner.py:72} INFO - Started process 15934 to run task
[2025-09-12T19:10:24.579+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.send_forecast scheduled__2025-09-11T16:00:00+00:00 [running]> on host b37bbfb35b9c
[2025-09-12T19:10:25.030+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='send_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-11T16:00:00+00:00'
[2025-09-12T19:10:25.032+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-12T19:10:28.841+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-12T19:10:28.924+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-12T19:10:28.925+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=send_forecast, run_id=scheduled__2025-09-11T16:00:00+00:00, execution_date=20250911T160000, start_date=20250912T191024, end_date=20250912T191028
[2025-09-12T19:10:29.026+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-12T19:10:29.133+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-12T19:10:29.157+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-09-12T19:10:29.266+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
